cfme/tests/cloud_infra_common/test_vm_instance_analysis.py

```diff
@@ -287,7 +321,7 @@ def detect_system_type(vm):
 
 
 @pytest.fixture(scope="module")
-def schedule_ssa(appliance, ssa_vm, ssa_profile, wait_for_task_result=True):
+def schedule_ssa(appliance, ssa_vm, ssa_profiled_vm, wait_for_task_result=True):
```

Comment:
Required: please remove ssa_vm in the tests when you use ssa_profiled_vm and update ssa_vm in the tests respectively

Reply:
@rlbabyuk I am working on it and will raise separate PR with all code refactored.

Reply:
Yes @rlbabyuk you are right, locally I have tested it.